client/payment-methods-map.js

```diff
@@ -78,4 +79,14 @@ export default {
 		Icon: IdealIcon,
 		currencies: [ 'EUR' ],
 	},
+	p24: {
+		id: 'ideal',
```

Comment:
```suggestion
		id: 'p24',
```

Reply:
I knew I forgot something😆 #realtalk: I fixed so many of these mistakes of mine before pushing 😅

Reply:
I knew I forgot something😆 #realtalk: I fixed so many of these mistakes of mine before pushing 😅 
Thanks for catching that!